src/FixedDataTableCell.js

```diff
@@ -146,6 +146,7 @@ class FixedDataTableCell extends React.Component {
     reorderCellLeft = Math.max(reorderCellLeft, 0);
     reorderCellLeft = Math.min(reorderCellLeft, farthestPossiblePoint);
 
+    // Todo: nextProps are supposed to be readonly
     // check if current cell belongs to the column that's being reordered
     if (nextProps.columnKey === nextProps.columnReorderingData.columnKey) {
```

Comment:
Throws error in dev mode the existing code is trying to change the nextProps which is readonly according to React. This needs some refactoring @pradeepnschrodinger .